docs/tutorials/multimodal/object_detection/quick_start/index.rst

```diff
@@ -4,7 +4,7 @@ Quick Start
 
    .. card::
       :title: Quick Start on a Tiny COCO Format Dataset
```

Comment:
Is this for the Tiny Motorbike dataset? Do we want to add the dataset name in the title